content/reference/v1beta1/repeatable.md

```diff
@@ -0,0 +1,304 @@
+
+The `yamlPath` field of the `template` must denote index position for arrays using square brackets.  For example, `spec.ports[0]` to select the first port entry for duplication.  All duplicate YAML will be appended to the final array in the `yamlPath`.
+
+`yamlPath` **must** end with an array.
+```yaml
+    template:
+    - apiVersion: v1
+      kind: Service
+      name: my-service
+      namespace: my-app
+      yamlPath: 'spec.ports[0]'
+```
+
+If the `yamlPath` field is blank, the entire YAML document matching the `template` will be replaced with a copy for each of the repeatable item entries.  The `metadata.name` field of the new doc will reflect the repeatable item `key`.
+
+## Templating
+
+The repeat items are called with the delimeters `repl[[ .itemName ]]` or `[[repl .itemName ]]`.  These delimiters can be placed anywhere inside of the `yamlPath` target node.
+```yaml
+    - port: repl{{ ConfigOption "[[repl .service_port ]]" | ParseInt }}
+      name: '[[repl .service_port ]]'
+```
+This repeatable templating is not compatible with sprig templating functions.  It is designed for inserting repeatable `keys` into the manifest. Repeatable templating **can** be placed inside of Replicated config templating.
+
+## Ordering
+
+Repeatable templates are processed before config template rendering.
+
+Repeatable items are processed in order of the template targets in the Config Spec file.  Effectively, this ordering is from the top of the Config Spec, by Config Group, by Config Item, and then by template target.
+
+```yaml
+    - name: ports
+      items:
+      - name: service_port
+        title: Service Port
+        type: text
+        repeatable: true
+        template:
+        - apiVersion: v1 #processed first
+          kind: Service
+          name: my-service
+          namespace: my-app
+          yamlPath: 'spec.ports[0]'
+        - apiVersion: v1 #processed second
+          kind: Service
+          name: my-service
+          namespace: my-app
+          yamlPath:
+        {other item properties ...}
+      - name: other_ports
+        title: Other Service Port
+        type: text
+        repeatable: true
+        template:
+        - apiVersion: v1 #processed third
+          kind: Service
+          name: my-other-service
+          namespace: my-app
+        {other item properties ...}
+    - name: deployments
+      items:
+      - name: deployment-name
+        title: Deployment Names
+        type: text
+        repeatable: true
+        template:
+        - apiVersion: apps/v1 #processed fourth
+          kind: Deployment
+          name: my-deployment
+          namespace: my-app
+          yamlPath:
+        {other item properties ...}
+```
+
+# Examples
+
+In this example, the default service port of "80" is included with the release. Port 443 is added as an additional port on the KOTSADM `Config` page, which is stored in the ConfigValues file.
```

Comment:
```suggestion
In these examples, the default service port of "80" is included with the release. Port 443 is added as an additional port on the KOTS Admin Console `Config` page, which is stored in the ConfigValues file.
```
Make the beginning plural because there are two examples. And current style is to write KOTS or KOTS Admin Console rather than kotsadm, except when that name must be used, like in code.